packages/app/src/__tests__/NexusPack.test.tsx

```diff
@@ -116,30 +116,35 @@ test('visualize NXentry group with implicit default child NXdata group', async (
   expect(tabs[0]).toHaveTextContent(NexusVis.NxSpectrum);
 });
 
-test('show error when encountering malformed NeXus metadata', async () => {
+test('show error when `default` entity is not found', async () => {
```

Comment:
I've split the `nexus_malformed` tests, since the `no_signal` one no longer shows an error. I've also removed the `default_not_string` test (and mock entity), as I no longer think it's worth having given how unlikely it is to occur.